simpleimage/make_rootfs.sh

```diff
@@ -390,6 +423,78 @@ EOF
 		rm -f "$DEST"/etc/ssh/ssh_host_*
 		do_chroot ln -s /run/resolvconf/resolv.conf /etc/resolv.conf
 		;;
+	opensuse)
+		cat > "$DEST/etc/resolv.conf" << "EOF"
+nameserver 8.8.8.8
```

Comment:
opensuse afaik does not auto-generate resolv.conf or have the resolvconf tool available